backend/src/routes/signup.ts

```diff
@@ -0,0 +1,30 @@
+import express, {Request, Response, NextFunction} from "express";
+import { User } from "../models/User"
+const router = express.Router();
+
+// add new user data to the mongoDB database
+router.post(
+    "/api/users/signup", 

```

Comment:
also make sure that this folder should be in /routes/api/users/signup.ts not /routes/signup.ts